src/devices/src/virtio/vsock/device.rs

```diff
@@ -44,54 +52,155 @@ use super::{defs, defs::uapi, EpollConfig};
 const AVAIL_FEATURES: u64 =
     1 << uapi::VIRTIO_F_VERSION_1 as u64 | 1 << uapi::VIRTIO_F_IN_ORDER as u64;
 
-pub struct Vsock<B: VsockBackend> {
+pub struct Vsock<B: VsockBackend + 'static> {
```

Comment:
nit: maybe add a `TODO` to remove this `'static` qualifier after #1540 goes in.

Reply:
I think they might not be related, but I did remove the bound here (it turns out trait bounds don't really do anything when defining a `struct`).